cruise.umple/src/class/UmpleInternalParser_CodeClass.ump

```diff
@@ -1328,7 +1327,7 @@ private Boolean checkIsDistributed(UmpleInterface uInterface)
     }
     if ( classToken.is("classDefinition") && "external".equals(aClass.getModifier()) )
       aClass.setModifier(""); // Remove the external modifier if a non-external specification of this class is found.
-
+    
```

Comment:
Best to avoid unneeded changes in spacing ... no big deal, but something to watch out for

Reply:
Thank you, I reverted them :)